smoke.py

```diff
@@ -87,6 +91,15 @@ def __del__(self):
 class BaseRuntimeSmokeTest(Test):
 
     def setUp(self):
+        compdir = self.params.get('compiler-test-dir')
+        if compdir is None:
+            self.error("Compiler test resource directory must be supplied")
```

Comment:
Good point. My latest commit makes the explicit command line parameters optional and uses the default location if not specified. I also made the the docker container image name an optional parameter while I was at it.